### PR TITLE
Fix some small errors in UG, LoanRate class and Email class.

### PR DESCRIPTION
**Pull request by Kelly9373**
Fix some prefix errors in UG. Update the message in LoanRate class for issue: https://github.com/CS2103-AY1819S1-F10-2/main/issues/205. Fix Gmail censoring problem for issue: https://github.com/CS2103-AY1819S1-F10-2/main/issues/174

### CODE REVIEW

File: src/main/java/loanbook/model/loan/LoanRate.java [hunk 1]
```diff
@@ -7,7 +7,8 @@
 public class LoanRate extends DataField<Double> {
 
     public static final String MESSAGE_LOANRATE_CONSTRAINTS =
-            "Rate should be not less than 0 and have at most 2 decimal places.";
+            "Rate should be not less than 0 and have at most 2 decimal places."
+                    + " Its first place cannot be 0, if the number is not 0 or 0 with decimal places.";
```
Comment (wn96) [inline src/main/java/loanbook/model/loan/LoanRate.java]: This sentence is extremely unclear.

File: src/main/java/loanbook/model/loan/Email.java [hunk 2]
```diff
@@ -38,6 +38,10 @@ public Email(String email) {
      * Check if user's new {@code Email} is a valid Gmail.
      */
     public static boolean isValidGmail(String email) {
+        if (email.length() < 16) {
```
Comment (wn96) [inline src/main/java/loanbook/model/loan/Email.java]: Why is 16 an invalid gmail?

Comment (Kelly9373) [reply src/main/java/loanbook/model/loan/Email.java]: Gmail's local part should have at least 6 chars, 6 + length of "@gmail.com" is 16, so the length of a valid gmail should not be less than 16.

Comment (wn96) [reply src/main/java/loanbook/model/loan/Email.java]: Why is this check necessary?

Comment (Kelly9373) [reply src/main/java/loanbook/model/loan/Email.java]: If the user set a user email like "123@gmail.com", the app will accept it, but this gmail does not follow the format of Gmail.

Comment (wn96) [reply src/main/java/loanbook/model/loan/Email.java]: Should we write a comment to explain? Also, Can this be done in the regex below?

File: src/main/java/loanbook/model/loan/LoanRate.java (hunk 1 above)
Comment (Kelly9373) [reply src/main/java/loanbook/model/loan/LoanRate.java]: Fixed.

File: src/main/java/loanbook/model/loan/Email.java (hunk 2 above)
Comment (Kelly9373) [reply src/main/java/loanbook/model/loan/Email.java]: Fixed.

**Pull request merged**